src/Controls/GlobeControls.js

```diff
@@ -516,65 +513,125 @@ class GlobeControls extends THREE.EventDispatcher {
             lastQuaternion.copy(this.camera.quaternion);
         }
         // Launch animationdamping if mouse stops these movements
-        if (this.enableDamping && this.state === this.states.ORBIT && this.player.isStopped() && (sphericalDelta.theta > EPS || sphericalDelta.phi > EPS)) {
+        if (this.enableDamping && state === this.states.ORBIT && this.player.isStopped() && (sphericalDelta.theta > EPS || sphericalDelta.phi > EPS)) {
+            this.player.setCallback(() => { this.update(this.states.ORBIT); });
             this.player.playLater(durationDampingOrbital, 2);
         }
     }
 
-    onMouseMove(event) {
-        if (this.player.isPlaying()) {
-            this.player.stop();
+    onStateChange(event) {
+        if (this.states.currentState === this.states.NONE) {
+            this.handleEndMovement(event);
+            return;
         }
-        if (this.enabled === false) { return; }
 
-        event.preventDefault();
-        const coords = this.view.eventToViewCoords(event);
+        CameraUtils.stop(this.view, this.camera);
+        this.player.stop();
+        this.onEndingMove();
+        if (this.enabled === false) { return; }
```

Comment:
could you move `this.enable`  in `statesControl` ?

Reply:
if `statesControls.enable` changes to `false`, so, change `currentState` to `NONE`, WDYT?

Reply:
When switching `enable` to false, `StateControl.onMouseUp` must be called to remove some pointer event listeners. The method also switches `currentState` to `NONE`.